interfaces/cython/cantera/ck2yaml.py

```diff
@@ -1328,6 +1336,33 @@ def parse_expression(expression, dest):
 
         return reaction, revReaction
 
+    def load_extra_file(self, path):
+        """
+        Load YAML-formatted entries from ``path`` on disk.
+        """
+        with open(path, 'rt', encoding="utf-8") as stream:
+            yml = yaml.round_trip_load(stream)
+
+        # do not overwrite reserved field names
+        reserved = {'generator', 'input-files', 'cantera-version', 'date',
+                    'units', 'phases', 'species', 'reactions'}
+        reserved &= set(yml.keys())
+        if reserved:
+            raise InputError("A YAML file provided as '--extra' input "
+                "(i.e. '{}') must not redefine reserved field names: "
+                "{}".format(path, reserved))
+
+        # replace header lines
+        if 'description' in yml:
+            if isinstance(yml['description'], str):
+                self.header_lines = yml.pop('description').split('\n')
```

Comment:
I think we should we consider appending to the header, if one also exists from the CK input file. Why did you prefer overwriting here?

Reply:
@bryanwweber ... that's a legitimate alternative and I'll likely adopt it. 

One issue is that there are some remaining white spaces on the left in the parsed header lines from CHEMKIN input (e.g. one blank in `gri30.yaml` ... i.e. the YAML indent is 2, but there are 3 spaces). It's a minor annoyance, which may not be worth fixing.

Reply:
You could `lstrip('! )` the header lines from the CK input, but there may be "significant" white space, in the sense that someone may have indented some lines for clarity or something. I agree, I'm not sure it's worth fixing unless it leads to incorrect YAML syntax.

Reply:
@bryanwweber ... wasn't too hard to strip the leading white spaces. I'm now appending the `description` as requested.